nautobot/extras/tests/test_serializers.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models import Q
+from django.test import override_settings
+from django.test import TestCase
+
+from nautobot.extras.api.serializers import(
+  ConfigContextSerializer
+)
+
+class ConfigContextSerializerTestCase(TestCase):
+  @override_settings(CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED=True)
+  def test_with_dynamic_groups_enabled(self):
+    serializer = ConfigContextSerializer()
+    self.assertIsNotNone(serializer.fields.get("dynamic_groups", None))
+
+  @override_settings(CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED=False)
+  def test_without_dynamic_groups_enabled(self):
+    serializer = ConfigContextSerializer()
```

Comment:
Can you please just roll this into `test_api.py::ConfigContextTest`. We don't have a precedent for testing serializers distinctly and it'll be nice to jus have these test methods included with the rest of the API tests in that class. Thanks!